src/PantheonTerminalWindow.vala

```diff
@@ -108,7 +108,7 @@ namespace PantheonTerminal {
 
         public PantheonTerminalWindow.with_working_directory (PantheonTerminalApp app, string location,
                                                               bool should_recreate_tabs = true) {
-            init (app, should_recreate_tabs);
+            init (app, should_recreate_tabs, true, false);
```

Comment:
Instead of doing this can we add a new public property? This conflicts with the changes in #185 to make this class more in GObject style

Reply:
I have matched the style of your pull request, this does still conflict but it is a pretty minor conflict to resolve.